packages/firestore/lib/src/firestore.dart

```diff
@@ -0,0 +1,53 @@
+// Copyright 2017, the Flutter project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+part of firebase_firestore;
+
+/// The entry point for accessing a Firebase Database. You can get an instance
```

Comment:
Is Firebase Database correct? Or should this be the Firestore?

Reply:
It's a mistake, thanks.